Dockerfile.art

```diff
@@ -43,5 +43,6 @@ USER 1001
 
 COPY --from=web-builder /usr/src/app/web/dist /opt/app-root/web/dist
 COPY --from=go-builder $REMOTE_SOURCES_DIR/cachito-gomod-with-deps/app/plugin-backend /opt/app-root
+COPY config/ /opt/app-root/config
```

Comment:
This should be copied from the web builder step

Reply:
Is there a reason to first copy it into the web-builder and then into the final image, rather than just putting it directly into the final image?

Reply:
no specific reason, we can leave it as it is